ontology/README.md

```diff
@@ -52,3 +52,40 @@ me:example:some_gene rdfs:seeAlso ENSEMBL:ESG00000XXXX . #and then pick up the a
 
 At this moment VG RDF wants a fully embedded variation graph. e.g. all positions in vg json have a single edit which covers a whole node. This is to enable easy SPARQL queries where substring operations are rarely used.
 
+
+## Annotations on pantograph
```

Comment:
@dimatr In case you started: This would be an example of a `.ttl`. Each entry can be dumped into any chunked file. The sequence of the entries does not matter.